From 5009252fba13a69b9962a8ac38b909c60ea1af17 Mon Sep 17 00:00:00 2001
From: David Frey <david.frey@sensirion.com>
Date: Tue, 14 Jul 2015 15:57:36 +0200
Subject: [PATCH 127/171] config: Enable SHT drivers for raspberry pi

The SHT temperature and humidity sensors are often used in weather
station projects.

Signed-off-by: David Frey <david.frey@sensirion.com>
---
 arch/arm/configs/bcm2709_defconfig | 4 +++-
 arch/arm/configs/bcmrpi_defconfig  | 4 +++-
 2 files changed, 6 insertions(+), 2 deletions(-)

--- a/arch/arm/configs/bcm2709_defconfig
+++ b/arch/arm/configs/bcm2709_defconfig
@@ -611,7 +611,9 @@ CONFIG_W1_SLAVE_BQ27000=m
 CONFIG_BATTERY_DS2760=m
 CONFIG_POWER_RESET=y
 CONFIG_POWER_RESET_GPIO=y
-# CONFIG_HWMON is not set
+CONFIG_HWMON=m
+CONFIG_SENSORS_SHT21=m
+CONFIG_SENSORS_SHTC1=m
 CONFIG_THERMAL=y
 CONFIG_THERMAL_BCM2835=y
 CONFIG_WATCHDOG=y
--- a/arch/arm/configs/bcmrpi_defconfig
+++ b/arch/arm/configs/bcmrpi_defconfig
@@ -604,7 +604,9 @@ CONFIG_W1_SLAVE_BQ27000=m
 CONFIG_BATTERY_DS2760=m
 CONFIG_POWER_RESET=y
 CONFIG_POWER_RESET_GPIO=y
-# CONFIG_HWMON is not set
+CONFIG_HWMON=m
+CONFIG_SENSORS_SHT21=m
+CONFIG_SENSORS_SHTC1=m
 CONFIG_THERMAL=y
 CONFIG_THERMAL_BCM2835=y
 CONFIG_WATCHDOG=y
